<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QMainWindow" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>876</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UAT Tool</string>
  </property>
  <widget class="QWidget" name="central_main">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>30</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>30</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stacked_widget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="pg_bugs">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>15</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="lbl_bugs">
          <property name="font">
           <font>
            <pointsize>28</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Bugs List</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="hlayout_filter_bugs" stretch="25,10,2,0,0">
          <property name="spacing">
           <number>10</number>
          </property>
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>150</number>
          </property>
          <item>
           <widget class="QLineEdit" name="le_search_bug">
            <property name="placeholderText">
             <string>Search by...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cb_search_bug">
            <item>
             <property name="text">
              <string>Service now ID</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Short description</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lbl_filterby_status">
            <property name="text">
             <string>Filter by:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cb_filter_status">
            <property name="placeholderText">
             <string>Status</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cb_filter_system">
            <property name="placeholderText">
             <string>System</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="tbl_bugs">
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustToContents</enum>
          </property>
          <property name="cornerButtonEnabled">
           <bool>false</bool>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pg_campaigns">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>15</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="lbl_campaigns">
          <property name="font">
           <font>
            <pointsize>28</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Test campaigns</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="tbl_campaigns"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pg_test_management">
       <layout class="QVBoxLayout" name="verticalLayout_12">
        <property name="spacing">
         <number>15</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="lbl_management">
          <property name="font">
           <font>
            <pointsize>28</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Test management</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTabWidget" name="tab_widget_management">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_cases">
           <attribute name="title">
            <string>Cases</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_13">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QTableView" name="tbl_cases"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_blocks">
           <attribute name="title">
            <string>Blocks</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_14">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QTableView" name="tbl_blocks"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pg_requirements">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>15</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="lbl_requirements">
          <property name="font">
           <font>
            <pointsize>28</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Requirements</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="hlayout_search_requirements" stretch="0,0,0">
          <property name="spacing">
           <number>10</number>
          </property>
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>150</number>
          </property>
          <item>
           <widget class="QLineEdit" name="le_search_requirement">
            <property name="placeholderText">
             <string>Search by ID...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lbl_filterby_system">
            <property name="text">
             <string>Filter by:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cb_system">
            <property name="placeholderText">
             <string>System</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="tbl_requirements"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pg_assets">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="spacing">
         <number>15</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="title_assets">
          <property name="font">
           <font>
            <pointsize>28</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Assets management</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTabWidget" name="tab_widget_assets">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_email">
           <attribute name="title">
            <string>Emails</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_15">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QTableView" name="tbl_emails"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_operators">
           <attribute name="title">
            <string>Operators</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QTableView" name="tbl_operators"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_drones">
           <attribute name="title">
            <string>Drones</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QTableView" name="tbl_drones"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_uas_zones">
           <attribute name="title">
            <string>UAS Zones</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QTableView" name="tbl_uas_zones"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_uhub_orgs">
           <attribute name="title">
            <string>U-hub Organizations</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QTableView" name="tbl_uhub_orgs"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_uhub_users">
           <attribute name="title">
            <string>U-hub Users</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QTableView" name="tbl_uhub_users"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_uspaces">
           <attribute name="title">
            <string>U-spaces</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_11">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QTableView" name="tbl_uspaces"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="hlayout_btn">
      <property name="leftMargin">
       <number>475</number>
      </property>
      <item>
       <widget class="QPushButton" name="btn_start">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_add">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_edit">
        <property name="text">
         <string>Edit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_remove">
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menu_bar_main">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>876</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_bugs">
    <property name="title">
     <string>Bugs</string>
    </property>
    <addaction name="action_new_bug"/>
    <addaction name="action_view_bugs"/>
   </widget>
   <widget class="QMenu" name="menu_campaigns">
    <property name="title">
     <string>Campaigns</string>
    </property>
    <addaction name="action_new_campaign"/>
    <addaction name="action_view_campaigns"/>
   </widget>
   <widget class="QMenu" name="menu_management">
    <property name="title">
     <string>Test Management</string>
    </property>
    <widget class="QMenu" name="menu_new_test">
     <property name="title">
      <string>New...</string>
     </property>
     <addaction name="action_new_case"/>
     <addaction name="action_new_block"/>
    </widget>
    <addaction name="menu_new_test"/>
    <addaction name="action_view_management"/>
   </widget>
   <widget class="QMenu" name="menu_requirements">
    <property name="title">
     <string>Requirements</string>
    </property>
    <addaction name="action_new_requirement"/>
    <addaction name="action_view_requirements"/>
   </widget>
   <widget class="QMenu" name="menu_assets">
    <property name="title">
     <string>Assets</string>
    </property>
    <widget class="QMenu" name="menu_new_asset">
     <property name="title">
      <string>New asset</string>
     </property>
     <addaction name="action_new_email"/>
     <addaction name="action_new_operator"/>
     <addaction name="action_new_drone"/>
     <addaction name="action_new_uas_zone"/>
     <addaction name="action_new_uhub_organization"/>
     <addaction name="action_new_uhub_user"/>
     <addaction name="action_new_uspace"/>
    </widget>
    <addaction name="menu_new_asset"/>
    <addaction name="action_view_assets"/>
   </widget>
   <widget class="QMenu" name="menu_statistics">
    <property name="title">
     <string>Statistics</string>
    </property>
   </widget>
   <addaction name="menu_bugs"/>
   <addaction name="menu_campaigns"/>
   <addaction name="menu_management"/>
   <addaction name="menu_requirements"/>
   <addaction name="menu_assets"/>
   <addaction name="menu_statistics"/>
  </widget>
  <widget class="QStatusBar" name="status_bar"/>
  <action name="action_new_bug">
   <property name="text">
    <string>New bug</string>
   </property>
  </action>
  <action name="action_view_bugs">
   <property name="text">
    <string>View bugs</string>
   </property>
  </action>
  <action name="action_new_campaign">
   <property name="text">
    <string>New campaign</string>
   </property>
  </action>
  <action name="action_view_campaigns">
   <property name="text">
    <string>View campaigns</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>New test block</string>
   </property>
  </action>
  <action name="action_view_management">
   <property name="text">
    <string>View tests</string>
   </property>
  </action>
  <action name="action_new_requirement">
   <property name="text">
    <string>New requirement</string>
   </property>
  </action>
  <action name="action_view_requirements">
   <property name="text">
    <string>View requirements</string>
   </property>
  </action>
  <action name="action_view_assets">
   <property name="text">
    <string>View assets</string>
   </property>
  </action>
  <action name="action_new_email">
   <property name="text">
    <string>Email</string>
   </property>
  </action>
  <action name="action_new_operator">
   <property name="text">
    <string>Operator</string>
   </property>
  </action>
  <action name="action_new_drone">
   <property name="text">
    <string>Drone</string>
   </property>
  </action>
  <action name="action_new_uas_zone">
   <property name="text">
    <string>UAS Zone</string>
   </property>
  </action>
  <action name="action_new_uhub_organization">
   <property name="text">
    <string>U-hub organization</string>
   </property>
  </action>
  <action name="action_new_uhub_user">
   <property name="text">
    <string>U-hub user</string>
   </property>
  </action>
  <action name="action_new_uspace">
   <property name="text">
    <string>U-space</string>
   </property>
  </action>
  <action name="action_new_case">
   <property name="text">
    <string>Test case</string>
   </property>
  </action>
  <action name="action_new_block">
   <property name="text">
    <string>Test block</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
